src/lib/parser/core/PokemonSaveParser.ts

```diff
@@ -34,6 +37,17 @@ function createEffectiveConfig (config: GameConfig): EffectiveConfig {
   }
 }
 
+/**
+ * Memory addresses for Pokémon Emerald (USA) in mGBA
+ * From official mGBA pokemon.lua script
+ */
+const EMERALD_MEMORY_ADDRESSES = {
+  PARTY_DATA: 0x20244ec, // _party address from pokemon.lua
+  PARTY_COUNT: 0x20244e9, // _partyCount address from pokemon.lua
+  POKEMON_SIZE: 100, // Size of each Pokemon struct (0x64 bytes)
+  MAX_PARTY_SIZE: 6, // Maximum party size
+} as const
+
```

Comment:
intigrate this better in the config system we already have
pokemon site and party size should already be defined somewhere.
the needed memory addresses should be configured in our overwritable classes, so that other configs like quatzal can also easily add their memory addresses